File: app/src/main/java/com/art2cat/dev/moonlightnote/Controller/Moonlight/EditMoonlightFragment.java
package com.art2cat.dev.moonlightnote.Controller.Moonlight;

import com.art2cat.dev.moonlightnote.model.Moonlight;

/**
 * Created by art2cat
 * on 10/1/16.
 */

public class EditMoonlightFragment extends MoonlightDetailFragment {

    public static EditMoonlightFragment newInstance(Moonlight moonlight, int flag) {

        EditMoonlightFragment editMoonlightFragment = new EditMoonlightFragment();
        editMoonlightFragment.setArgs(moonlight, flag);
        return editMoonlightFragment;
    }
}

File: app/src/main/java/com/art2cat/dev/moonlightnote/controller/moonlight/EditMoonlightFragment.kt
package com.art2cat.dev.moonlightnote.controller.moonlight

import com.art2cat.dev.moonlightnote.model.Moonlight

/**
 * Created by Rorschach
 * on 24/05/2017 7:58 PM.
 */
class EditMoonlightFragment : MoonlightDetailFragment() {
    companion object {

        fun newInstance(moonlight: Moonlight, flag: Int): EditMoonlightFragment {

            val editMoonlightFragment = EditMoonlightFragment()
            editMoonlightFragment.setArgs(moonlight, flag)
            return editMoonlightFragment
        }
    }
}